libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -224,21 +248,34 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
         return !this.props.documentationUrl ? itemGroups : [...itemGroups, [this.getHelpMenuLink()]];
     };
 
-    private addSearchMenu = (itemGroups: IHeaderMenuItem[][]): IHeaderMenuItem[][] => {
-        if (!this.props.search) {
+    private addAdditionalItems = (itemGroups: IHeaderMenuItem[][]): IHeaderMenuItem[][] => {
+        const additionalItems = [];
+        if (this.props.search) {
+            additionalItems.push({
+                key: "gs.header.search",
+                className: "gd-icon-header-search",
+                onClick: this.toggleSearchMenu,
+            });
+        }
+
+        if (this.props.notificationsPanel) {
+            additionalItems.push({
+                key: "gs.header.notifications",
+                className: "gd-icon-header-notifications",
+                icon: (
+                    <span className="gd-header-notifications-icon">
+                        <Icon.Alert width={16} height={16} />
+                    </span>
+                ),
+                onClick: this.toggleNotificationsMenu,
+            });
+        }
+
+        if (!additionalItems.length) {
             return itemGroups;
         }
 
-        return [
-            ...itemGroups,
-            [
-                {
-                    key: "gs.header.search",
-                    className: "gd-icon-header-search",
-                    onClick: this.toggleSearchMenu,
-                },
-            ],
-        ];
+        return [...itemGroups, additionalItems];
```

Comment:
not sure, but should there be `...itemGroups, ...additionalItems` instead?

Reply:
it's same as previous code - it was not spread as well (its another section group)